### ISSUE TITLE
Text colors are not correct

### ISSUE THREAD

**Issue by cilerler** [mjbvz/vscode-markdown-mermaid#10]
reference:// knsv/mermaid#570

As you can see it in the screenshot, `ALT`,`OPT`,`LOOP` titles are not visible. _(text is there, it is just a dark color which makes it very difficult to read on dark theme.)_

![2017-09-06_22-07-10](https://user-images.githubusercontent.com/385958/30142632-ee85784c-934f-11e7-83c6-6004e01ccccc.png)

> _@tylerlong stated_
> _Mermaid has a dark theme which is suitable for dark backgrounds. https://unpkg.com/mermaid@7.0.0/dist/mermaid.dark.css_

**Comment by mjbvz** [comment]
Can you please share the source of that example. We already use `mermaid.dark.css` but I suspect some rule is being overridden

**Comment by cilerler** [comment]
```mermaid
sequenceDiagram
    participant Service
    participant Provider

    loop Minute query
        Service->> Provider: request Records

        alt is maintenance
            Provider --x Service: response No-Record
        else is page
            Provider --x Service: response Records

            Service ->> Service: locate New Records

            opt new Record exist
                loop Details
                    Service ->> Provider: request Details
                    Provider --x Service: respose Details
                end
            end
        end

        Note right of Service: Wait until<br/>time is up...

    end
```

**Comment by tylerlong** [comment]
I posted reply to https://github.com/knsv/mermaid/issues/570

Let me know if there is anything else that I can help.

**Comment by jnnnnn** [comment]
This was a bug in Mermaid itself. It was fixed in https://github.com/knsv/mermaid/pull/837 which was released in Mermaid 8.1.0. I will create a PR to update the version of mermaid.
